troubleshooting/known_issues/issues_with_ncn_health_checks.md

```diff
@@ -0,0 +1,125 @@
+#### Known issues with NCN health checks
```

Comment:
As per my comments above, this file needs to be structured into using headings so that we can directly link to specific sections as per type of check failure.